bpf/lib/l3.h

```diff
@@ -14,6 +14,17 @@
 #include "icmp6.h"
 #include "csum.h"
 
+/*
+ * When the host routing is enabled we need to check policies at source.
+ * Otherwise, as the skb will be delivered directly to pod's namespace,
+ * the ingress policy (the cil_to_container BPF program) will be bypassed.
```

Comment:
Who is "we" here? Which component(s) are checking policies at source?
Avoid future tense.
```suggestion
 * When the host routing is enabled we need to check policies at source.
 * Otherwise, the skb is delivered directly to pod's namespace and
 * the ingress policy (the cil_to_container BPF program) is bypassed.
```